tests/models/test_scripts.py

```diff
@@ -19,6 +19,34 @@ def test_cli_run_basic_gan(cli_args):
         cli_main()
 
 
+@pytest.mark.parametrize(
+    "cli_args",
+    [
+        f"--dataset mnist --scale_factor 4 --data_dir {DATASETS_PATH} --max_epochs 1"
+        " --batch_size 2 --limit_train_batches 2 --limit_val_batches 2"
```

Comment:
How about  adding `--fast_dev_run` for faster testing?
```suggestion
        f"--dataset mnist --scale_factor 4 --data_dir {DATASETS_PATH} --max_epochs 1"
        " --batch_size 2 --limit_train_batches 2 --limit_val_batches 2 --fast_dev_run 1"
```